src/selection_criteria.rs

```diff
@@ -126,6 +126,30 @@ pub struct ReadPreferenceOptions {
     /// specified for an operation, the operation will return an error.
     #[builder(default)]
     pub max_staleness: Option<Duration>,
+
+    /// Specifies hedging behavior for reads.
```

Comment:
I think we could include a little more contextual info here so that users won't always need to go to the external page. Specifically, I think we ought to mention that this only applies to sharded clusters in 4.4+ and that it is automatically enabled for "nearest" read preferences.

Reply:
Updated